UnityProject/Assets/Scripts/Items/MetalTrigger.cs

```diff
@@ -9,7 +9,7 @@
 public class MetalTrigger : PickUpTrigger
 {
 	private bool isBuilding;
-	public GameObject griderPrefab;
+	public GameObject girderPrefab;
```

Comment:
are you sure prefab didn't lose reference because of that change?
use [FormerlySerializedAs("griderPrefab")] above it to keep the references

Reply:
Well, MetalTrigger is only used for the metal prefab and I updated the reference there, so should I still add the FormerlySerializedAs (and should I remove the .prefab from the commit)

Reply:
just grab a piece of metal ingame and see if it works

Reply:
It does, I tested my commit